osu.Game/Overlays/Settings/Sections/Gameplay/SongSelectSettings.cs

```diff
@@ -26,18 +26,21 @@ private void load(OsuConfigManager config)
                 {
                     LabelText = "Show converted beatmaps",
                     Bindable = config.GetBindable<bool>(OsuSetting.ShowConvertedBeatmaps),
+                    Keywords = new[] { "convert" }
```

Comment:
Not necessary, since it's already in the label text. (That's why I used "converts", not "convert")

Reply:
Adding plurals feels completely backwards. I'd sooner advocate implementing a similar mechanism to stable where the search will stop filtering if it can see zero results would be returned.